src/javaScript/indexDom.js

```diff
@@ -103,35 +107,51 @@ function populateGamesContainer (payload, gamesContainer) {
 }
 
 function populateGamesHighlights (payload, mainHighlightContainer) {
-  // insert dom changes here
-  payload.forEach((game) => {
+  payload.forEach((game, index) => {
+    const gameplayImages = getImage(game, index)
     mainHighlightContainer.insertAdjacentHTML(
       'beforeend',
-      `<div
-    class="highlight-container group hidden h-full w-full max-w-sm border-2 border-white"
-  >
-      <img
-        class="h-full m w-full group-hover:hidden"
-        src="/assets/home-page/highlights/${game.highlight}"
-        alt="${game.name} cover"
-      />
-    <div class="hidden h-3/5 justify-between group-hover:flex">
-      screenshots
-    </div>
-    <p class="hidden h-1/5 text-center group-hover:block">${game.name}</p>
-    <div class="flex justify-between">
-      <div
-        class="dark-button mx-5 hidden size-10 items-center justify-center rounded-full ring-2 group-hover:flex"
-      >
-        🛒
-      </div>
-      <div
-        class="dark-button mx-5 hidden size-10 w-24 items-center justify-center rounded-full ring-2 group-hover:flex"
-      >
-        Price: $${game.price}
-      </div>
-    </div>
-`
+      `<div class="highlight-container group hidden h-full w-full max-w-sm border-4 border-[#0B1215]">
+          <!-- Game highlight image -->
+          <img class="h-full  w-full group-hover:hidden"
+               src="/assets/home-page/highlights/${game.highlight}"
+               alt="${game.name} cover"/>
+          <!-- Gameplay images -->
+          <div class="hidden h-3/5 justify-between group-hover:flex">
+            <div class="gameplay-image flex flex-1 justify-center m-3">
+            ${gameplayImages}
+            </div>
+          </div>
+          <!-- Game name -->
+          <p class="hidden h-1/5 text-center group-hover:block">${game.name}</p>
+          <!-- Add to cart button and price -->
+          <div class="flex justify-between">
+            <div class="dark-button mx-5 hidden size-10 items-center justify-center rounded-full ring-2 group-hover:flex">🛒</div>
+            <div class="dark-button mx-5 hidden size-10 w-24 items-center justify-center rounded-full ring-2 group-hover:flex">
+              Price: $${game.price}
+            </div>
+          </div>
+        </div>`
     )
   })
 }
+
+function getImage (game, index) {
+  let currentIndex = 0
+
+  if (Array.isArray(game.gameplay) && game.gameplay.length > 0) {
+    const gameplayImages = `<img class="w-full h-full gameplay-image-${index}" src="/assets/home-page/gameplay/${game.gameplay[currentIndex]}" alt="${game.name} gameplay moment"/>`
+
+    const updateImage = () => {
+      currentIndex = (currentIndex + 1) % game.gameplay.length
+      const imgElement = document.querySelector(`.gameplay-image-${index}`)
+      imgElement.src = `/assets/home-page/gameplay/${game.gameplay[currentIndex]}`
+    }
+
+    setInterval(updateImage, 2000)
```

Comment:
Unclear to me why is the setInterval needed.

Reply:
i dont have space to put all the gameplays in there so i rotate them when the user hovers over the container... 
Should i not do it?

Reply:
Is this valid for all screen sizes?